modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -263,6 +264,141 @@ Marketplace URL: https://marketplace.owncloud.com/apps/contacts[Contacts]
 
 For commands for managing contacts, please see the DAV Command section in the occ core command set.
 
+[[files_Antivirus]]
+Anti-Virus
+----------
+
+Marketplace URL: https://marketplace.owncloud.com/apps/files_antivirus[Anti-Virus]
+
+Use these commands to configure the Anti-Virus app.
```

Comment:
It makes more sense, to me, to link "the Anti-Virus app", instead of having the link text on line 271.

Reply:
But the way it is done is what you requested and how I implemented it and you approved ...
It renders like that:
![image](https://user-images.githubusercontent.com/3321281/46062812-4189a000-c16b-11e8-9fd0-1d7c411e883a.png)
Please advice

Reply:
When I was reading the text just now, it seemed that it would flow more naturally, as I suggested. But now that you mention it, I see that I was wrong.

Reply:
pfff  😄